tests/contrib/aiohttp/test_request_safety.py

```diff
@@ -20,6 +20,9 @@ class TestAiohttpSafety(TraceTestCase):
     bad traces are produced but the ``Context`` object will not
     leak memory.
     """
+    def setUp(self):
+        super().setUp()
+        asyncio.set_event_loop(self.loop)
```

Comment:
IIRC this is here because the old aiohttp test frameworks don't set the current event loop for the loop that it creates.